army-end-strength-budget/js/splinebar.js

```diff
@@ -102,16 +99,15 @@ $(function() {
         tickInterval: 1,
         allowDecimals: false
       },
-      // Y Axis
-      yAxis: [{ // Primary yAxis
+      yAxis: [{
           title: {
               text: 'Army End Strength',
           },
           labels: {
-              format: '{value}k'
+              format: '{value}K'
           },
           reversedStacks: false
-      }, { // Secondary yAxis
+      }, {
           title: {
               text: 'Discretionary & Mandatory Budget Authority<br/>(in FY19 Dollars)',
```

Comment:
<img width="140" alt="screen shot 2018-09-06 at 10 02 51 am" src="https://user-images.githubusercontent.com/1896496/45162506-2a830e00-b1bc-11e8-8287-7631808cd267.png">

Since we increased the size of the Y Axis labels, we can remove the line break from the title so the `(in FY19 Dollars)` is on the same line as `Authority`.

Reply:
Changed has been added and code pushed.